product.py

```diff
@@ -5,3 +5,58 @@
     :copyright: (c) 2015 by Openlabs Technologies & Consulting (P) Limited
     :license: BSD, see LICENSE for more details.
 """
+from trytond.model import ModelSQL, ModelView, fields
+from trytond.pool import PoolMeta
+from trytond.pyson import Eval
+
+__metaclass__ = PoolMeta
+__all__ = ['Attribute', 'AttributeOption', 'Product']
+
+
+class AttributeOption(ModelSQL, ModelView):
+    "Product Attribute Option"
+    __name__ = 'product.attribute.option'
+
+    name = fields.Char("Name", required=True)
+
+
+class Attribute:
+    "Attribute"
+    __name__ = 'product.attribute'
+
+    product = fields.Many2One("product.product", "Product", required=True)
+
+    attribute_option = fields.Many2One(
```

Comment:
This is not needed. From here `One2Many` to `product.attribute.strict_selection` visible only if the `type_` is strict_selection